packages/app-accounts/src/Editor.tsx

```diff
@@ -98,21 +100,39 @@ class Editor extends React.PureComponent<Props, State> {
     const { currentPair, defaultValue, editedName } = this.state;
 
     if (!currentPair) {
-      return t('editor.none', {
-        defaultValue: 'There are no saved accounts. Add some first.'
-      });
+      return (
+        <div>
+          <div>{t('editor.nosaved', { defaultValue: 'There are no saved accounts. Create an account or upload a JSON file of a saved account.' })}</div>
+          <div className={'accounts--Address-wrapper'}>
+            <div className={'accounts--Address-file'}>
+              <UploadButton
+                size='big'
```

Comment:
Since this is only used in a single place - do we need a `size` prop? Set that as the default in the component.